[2024-07-02T04:52:31.571+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-02T04:52:31.590+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v03.postgres_to_s3 scheduled__2022-10-06T00:00:00+00:00 [queued]>
[2024-07-02T04:52:31.599+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v03.postgres_to_s3 scheduled__2022-10-06T00:00:00+00:00 [queued]>
[2024-07-02T04:52:31.600+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-02T04:52:31.616+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-10-06 00:00:00+00:00
[2024-07-02T04:52:31.625+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=23574) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-02T04:52:31.625+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgers_hooks_v03', 'postgres_to_s3', 'scheduled__2022-10-06T00:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_hook.py', '--cfg-path', '/tmp/tmpp7tabe6h']
[2024-07-02T04:52:31.627+0000] {standard_task_runner.py:63} INFO - Started process 23579 to run task
[2024-07-02T04:52:31.627+0000] {standard_task_runner.py:91} INFO - Job 176: Subtask postgres_to_s3
[2024-07-02T04:52:31.673+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgers_hooks_v03.postgres_to_s3 scheduled__2022-10-06T00:00:00+00:00 [running]> on host 468cd84e5e4e
[2024-07-02T04:52:31.790+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgers_hooks_v03' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-06T00:00:00+00:00'
[2024-07-02T04:52:31.791+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-02T04:52:31.809+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:212 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-07-02T04:52:31.819+0000] {base.py:84} INFO - Using connection ID 'postgres_conn_id' for task execution.
[2024-07-02T04:52:31.826+0000] {dag_with_postgress_hook.py:34} INFO - Saved orders data in text file: dags/get_orders_20221006.txt
[2024-07-02T04:52:31.834+0000] {base.py:84} INFO - Using connection ID 'minio_s3_conn' for task execution.
[2024-07-02T04:52:31.835+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2024-07-02T04:52:32.380+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-02T04:52:32.381+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-02T04:52:32.394+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgers_hooks_v03, task_id=postgres_to_s3, run_id=scheduled__2022-10-06T00:00:00+00:00, execution_date=20221006T000000, start_date=20240702T045231, end_date=20240702T045232
[2024-07-02T04:52:32.523+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-02T04:52:32.549+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-02T04:52:32.553+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
